ts/utils/paymentMethod.ts

```diff
@@ -49,6 +52,8 @@ export const getPaymentMethodHash = (
   switch (pm.kind) {
     case "Satispay":
       return pm.info.uuid;
+    case "PayPal":
+      return pm.info.emailPp;
```

Comment:
Is it right that the hash is the plain email for future capabilities related with paypal?

Reply:
It's a good question!
At the moment the payment hash is almost used for 

- BPD, and I guess paypal won't be join it.
- onboarding phase where you have to pick back a previous selected payment method (i.e bancomat or bancomat pay)

I believe both of those scenario won't include Paypal.
Furthermore Paypal is identified using the account email.
Anyway I will forward this question to those people that know more about it

Since the hash for Paypal have no utilities, at the moment, I added it because otherwise I got a compile error due to the fact I add a new payment method type

Reply:
Just wanted to make sure it was considered for future implementations that may require a hash!